src/modules/transport-mode/filter/filter.module.css

```diff
@@ -38,7 +52,8 @@
   display: flex;
   gap: token('spacing.small');
   align-items: center;
-  height: 1.75rem;
+  border: 1px solid token('color.background.neutral.3.background');
+  height: 2.7rem;
```

Comment:
This makes the transport mode boxes to be 43.2px tall – in the design they look like they should be 36px. `2.25rem` might look like is closer to that.